components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/Utils.java

```diff
@@ -29,6 +29,9 @@
 import org.apache.synapse.rest.RESTConstants;
 import org.json.JSONArray;
 import org.json.JSONObject;
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
```

Comment:
shall we add these to bundle imports as well

Reply:
Fixed in 8d125b0b73b6f415386fdfeb92c6212426f04832